src/EasyList/TodoMenu.cs

```diff
@@ -1,138 +1,100 @@
 ﻿using EasyList.DataModels;
 using EasyList.Enums;
-using EasyList.Factories;
-using EasyList.Interfaces;
 using Sharprompt;
 using System;
 using System.Collections.Generic;
-using System.Linq;
 
 namespace EasyList
 {
     public class TodoMenu
     {
-        private static IEnumerable<string> ValidateAdd(Dictionary<string, string> input)
-        {
-            if (string.IsNullOrWhiteSpace(input["label"]))
-            {
-                yield return "Label cannot be empty";
-            }
-            if (DateTimeOffset.TryParse(input["duedate"], out DateTimeOffset tempDate))
-            {
-                if (tempDate < DateTime.UtcNow)
-                {
-                    yield return "Due date cannot be in the past";
-                }
-            }
-        }
-
-        private static bool Validate(TODOMENU command, Dictionary<string, string> input)
-		{
-            var errors = command switch {
-                TODOMENU.Add => ValidateAdd(input),
-                // register the rest of the options that need to be validated here
-                _ => throw new InvalidOperationException($"No validator exists for {command}"),
-            };
-            
-            if (errors.Any())
-            {
-                Console.WriteLine("The input has the following errors:");
-                foreach (var error in errors)
-                {
-                    Console.WriteLine("\t"+error);
-                }
-
-                return false;
-            }
-
-            return true;
-        }
 
         public static void Run()
         {
-            ITodoService _todoService = Factory.CreateTodoServiceDB();
-            while(true)
+            var _todoValidate = new Validate();
```

Comment:
The class is Todo specific and should be called TodoValidator, i think a name for the field should be the same name as the class